go.mod

```diff
@@ -5,65 +5,65 @@ go 1.20
 require (
 	github.com/go-logr/logr v1.2.4
 	go.uber.org/zap v1.24.0
-	k8s.io/api v0.26.3
-	k8s.io/apimachinery v0.26.3
-	k8s.io/client-go v0.26.3
-	sigs.k8s.io/controller-runtime v0.14.6
+	k8s.io/api v0.27.2
```

Comment:
Should these version updates be included in this PR?

Reply:
There was a merging error. I have rolled this back to 0.26.3. The 0.27.2 PR will come later.